functions/ts/kubeval/forked/github.com/instrumenta/openapi2jsonschema/command.py

```diff
@@ -0,0 +1,242 @@
+#!/usr/bin/env python
```

Comment:
Can you explain why we need to run this utility? Can't kubeval already work with the OpenAPI document served by a cluster?

Reply:
> Can't kubeval already work with the OpenAPI document served by a cluster?

Unfortunately no.
`kubeval` by default fetches json schemas from kubernetesjsonschema.dev which is a static site with its content in https://github.com/instrumenta/kubernetes-json-schema. [Here](https://github.com/instrumenta/kubernetes-json-schema/blob/master/build.sh#L178) is the script to generate new contents. The script is using `openapi2jsonschema`  to do the conversion. https://github.com/instrumenta/openapi2jsonschema is not very active any more, so I fork it and fix a bug and make some improvement to make the conversion faster.